File: web/src/main/java/net/yamac/springboot2/example/ExampleController.java
package net.yamac.springboot2.example;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RequiredArgsConstructor
@RestController
public class ExampleController {

    private final ExampleService exampleService;

    @GetMapping("/")
    public String getIndex() {
        return exampleService.getMessage();
    }
}

File: web-kotlin/src/main/kotlin/net/yamac/springboot2/example/kotlin/ExampleController.kt
package net.yamac.springboot2.example.kotlin

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class ExampleController(val exampleService: ExampleService) {

    @GetMapping("/")
    fun getIndex(): String = exampleService.getMessage()
}
